chromium_src/net/tools/transport_security_state_generator/input_file_parsers.cc

```diff
@@ -0,0 +1,326 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#define ParseJSON ParseJSON_ChromiumImpl
+#define ParseCertificatesFile ParseCertificatesFile_ChromiumImpl
+#include "../../../../../net/tools/transport_security_state_generator/input_file_parsers.cc"
+#undef ParseCertificatesFile
+#undef ParseJSON
+
+namespace net {
+
+namespace transport_security_state {
+
+bool ParseCertificatesFile(base::StringPiece certs_input, Pinsets* pinsets) {
+  base::StringPiece brave_certs = R"brave_certs(TestSPKI
+sha256/AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=
```

Comment:
string literals have a max length of 65,536. I think we're ok for now but we'll probably need to change this eventually. Have you considered using a pak file and loading the resource from there?

Reply:
Similar to:
`std::string str = ui::ResourceBundle::GetSharedInstance().GetRawDataResource(
    IDR_BRAVE_TAG_MANAGER_POLYFILL).as_string();`

Reply:
This runs at build time so pak wouldn't work. I thought about loading from a file, but paths get a little weird so I just used a string for now